### ISSUE TITLE
biwi_stage2.pth.tar significant loss in training dataset.

### ISSUE THREAD

**Issue by youngstu** [Doubiiu/CodeTalker#28]
I used the pretrained model(biwi_stage2.pth.tar) for fine-tuning training and found that there were significant loss in both the training and validation dataset, which is a bit unreasonable. Was biwi_stage2.pth.tar trained in the testing set Or there are some additional techniques outside of the code. Could you help explain? Very Thanks.
